packages/suite/src/components/guide/HeaderBreadcrumb.tsx

```diff
@@ -0,0 +1,137 @@
+    const analytics = useAnalytics();
+
+    const { language, indexNode, currentNode } = useSelector(state => ({
+        language: state.suite.settings.language,
+        indexNode: state.guide.indexNode,
+        currentNode: state.guide.currentNode,
+    }));
+
+    const { setView, openNode } = useActions({
+        setView: guideActions.setView,
+        openNode: guideActions.openNode,
+    });
+
+    // if no parent available, offer navigation to guide dashboard
+    const fallbackBreadcrumb = (
+        <BreadcrumbWrapper>
+            <CategoryLink onClick={() => setView('GUIDE_DEFAULT')}>
+                <Translation id="TR_GUIDE_DASHBOARD" />
+            </CategoryLink>
+        </BreadcrumbWrapper>
+    );
+
+    if (!currentNode || !indexNode) return fallbackBreadcrumb;
+
+    const parentNodes = findAncestorNodes(currentNode, indexNode).filter(
+        node => node.type === 'category',
+    ) as Category[];
+
+    if (!parentNodes.length) return fallbackBreadcrumb;
+
+    const navigateToCategory = (node: Category) => {
+        openNode(node);
+        analytics.report({
+            type: 'guide/header/navigation',
+            payload: {
+                type: 'category',
+                id: node.id,
+            },
+        });
+    };
+
+    const navigateToGuideDashboard = () => {
+        setView('GUIDE_DEFAULT');
+        analytics.report({
+            type: 'guide/header/navigation',
+            payload: {
+                type: 'category',
+                id: '/',
+            },
+        });
+    };
+
+    // If page is part of level 1 category, breadcrumb should consist of Dashboard / Category Level 1
+    // If page is part of level 2 category, breadcrumb should consist of Category Level 1 / Category Level 2
+    const parentNode = parentNodes.pop();
+    const grandParentNode = parentNodes.pop();
+
+    return (
+        <BreadcrumbWrapper>
+            <PreviousCategoryLink
+                onClick={() => {
+                    if (grandParentNode) {
+                        navigateToCategory(grandParentNode);
+                    } else {
+                        navigateToGuideDashboard();
+                    }
+                }}
+            >
+                {grandParentNode ? (
+                    getNodeTitle(grandParentNode, language)
+                ) : (
+                    <Translation id="TR_GUIDE_DASHBOARD" />
+                )}
+            </PreviousCategoryLink>
+            <BreadcrumbDelimiter>/</BreadcrumbDelimiter>
+            <CategoryLink
+                onClick={() => {
+                    if (grandParentNode) {
+                        navigateToCategory(grandParentNode);
+                    } else if (parentNode) {
+                        navigateToCategory(parentNode);
+                    }
+                }}
+            >
+                {
+                    // @ts-ignore parentNode is always defined
+                    getNodeTitle(parentNode, language)
```

Comment:
I believe that it's almost never a good idea to ignore typescript hints... Especially if it's easy to add the check here 

```suggestion
                    parentNode  && getNodeTitle(parentNode, language)
```

Reply:
Done